src/firefly/js/visualize/ui/DrawLayerPanel.jsx

```diff
@@ -21,16 +21,12 @@ export const DRAW_LAYER_POPUP= 'DrawLayerPopup';
 
 function getDialogBuilder() {
     var popup= null;
+    const defaultTitle = 'Drawing Layers';
     return () => {
         if (!popup) {
-            var title = 'Drawing Layers';
-            //try {
-            //    if(visRoot()){
-            //        title += ': '+getActivePlotView(visRoot()).plots[0].title;
-            //    }
-            //}catch (E){
-            //    //
-            //}
+            var title = get( getActivePlotView(visRoot()), ['plots', '0', 'title']);
```

Comment:
This is not the correct way to get the primary plot.  The plots array should never be access directly.  Beyond that 0 is not always the primary plot.
the `primePlot` function should always be called here.  

The code should be
```
const plot= primePlot(visRoot());
title= plot ? plot.title : null;
```